test/performance_tests.jl

```diff
@@ -146,6 +149,11 @@ function benchmark_test(FT)
     )
     bench_press(CMI_het.deposition_J, (kaolinite, Delta_a_w), 230)
     bench_press(CMI_het.ABIFM_J, (desert_dust, Delta_a_w), 230)
+    bench_press(
+        CMI_het.INP_concentration_frequency,
+        (ip_frostenberg, INPC, T_air_cold),
+        230,
```

Comment:
Did you check that `230` is time needed to run this in the CI?

Reply:
I checked it now and it looks like `110` is enough